Adding require-context.macro and calling requireContext broke Storybook

Hey guys, I was following Storybook tutorial (https://www.learnstorybook.com/react/en/simple-component/)

On installing `require-context.macro` and changing `require.context` to `requireContext` in `config.js` to be able do use Storyshot, everything broke.

I've already installed `babel-plugin-macros` as stated in the readme, but it's still not working.

Check out the errors:

```js
ERROR in ./node_modules/cosmiconfig/dist/readFile.js
Module not found: Error: Can't resolve 'fs' in '/Users/aurus/dev/storybook/taskbox/node_modules/cosmiconfig/dist'
 @ ./node_modules/cosmiconfig/dist/readFile.js 4:11-24
 @ ./node_modules/cosmiconfig/dist/createExplorer.js
 @ ./node_modules/cosmiconfig/dist/index.js
 @ ./node_modules/babel-plugin-macros/dist/index.js
 @ ./node_modules/require-context.macro/require-context.macro.js
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js ./node_modules/webpack-hot-middleware/client.js?reload=true

ERROR in ./node_modules/is-directory/index.js
Module not found: Error: Can't resolve 'fs' in '/Users/aurus/dev/storybook/taskbox/node_modules/is-directory'
 @ ./node_modules/is-directory/index.js 10:9-22
 @ ./node_modules/cosmiconfig/dist/getDirectory.js
 @ ./node_modules/cosmiconfig/dist/createExplorer.js
 @ ./node_modules/cosmiconfig/dist/index.js
 @ ./node_modules/babel-plugin-macros/dist/index.js
 @ ./node_modules/require-context.macro/require-context.macro.js
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js ./node_modules/webpack-hot-middleware/client.js?reload=true

ERROR in ./node_modules/resolve/lib/async.js
Module not found: Error: Can't resolve 'fs' in '/Users/aurus/dev/storybook/taskbox/node_modules/resolve/lib'
 @ ./node_modules/resolve/lib/async.js 2:9-22
 @ ./node_modules/resolve/index.js
 @ ./node_modules/babel-plugin-macros/dist/index.js
 @ ./node_modules/require-context.macro/require-context.macro.js
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js ./node_modules/webpack-hot-middleware/client.js?reload=true

ERROR in ./node_modules/resolve/lib/sync.js
Module not found: Error: Can't resolve 'fs' in '/Users/aurus/dev/storybook/taskbox/node_modules/resolve/lib'
 @ ./node_modules/resolve/lib/sync.js 2:9-22
 @ ./node_modules/resolve/index.js
 @ ./node_modules/babel-plugin-macros/dist/index.js
 @ ./node_modules/require-context.macro/require-context.macro.js
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js ./node_modules/webpack-hot-middleware/client.js?reload=true

ERROR in ./node_modules/import-fresh/node_modules/resolve-from/index.js
Module not found: Error: Can't resolve 'module' in '/Users/aurus/dev/storybook/taskbox/node_modules/import-fresh/node_modules/resolve-from'
 @ ./node_modules/import-fresh/node_modules/resolve-from/index.js 3:15-32
 @ ./node_modules/import-fresh/index.js
 @ ./node_modules/cosmiconfig/dist/loaders.js
 @ ./node_modules/cosmiconfig/dist/index.js
 @ ./node_modules/babel-plugin-macros/dist/index.js
 @ ./node_modules/require-context.macro/require-context.macro.js
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js ./node_modules/webpack-hot-middleware/client.js?reload=true
```
The funny part is that `yarn test` runs ok... and `Storyshots` seems to work passing all tests, but Storybook itself don't.

What should be done?

Thanks

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # require-context.macro
 
-A macro needed for some advanced Storybook setups
+A macro needed for some advanced Storybook@4 and Storybook@3 setups
 
 [![Babel Macro](https://img.shields.io/badge/babel--macro-%F0%9F%8E%A3-f5da55.svg?style=flat-square)](https://github.com/kentcdodds/babel-plugin-macros)
 
@@ -8,4 +8,41 @@ Credit to [the original plugin](https://github.com/smrq/babel-plugin-require-con
 
 ## Usage
 
-Ensure you have `babel-plugin-macros` installed within your project. Afterwards, simply import this function and call it in place of `require.context()`.
+Ensure you have `babel-plugin-macros` installed within your project.
+
+`yarn add -D babel-plugin-macros`
+
+Then install this specific macro
+
+`yarn add -D require-context.macro`
+
+Afterwards, simply import this function and call it in place of `require.context()` inside
+`.storybook/config.js`.
+
+```javascript
+// .storybook/config.js
+
+import { configure } from '@storybook/react';
+import requireContext from 'require-context.macro';
+
+import '../src/index.css';
+
+const req = requireContext('../src/components', true, /\.stories\.js$/);
+
+function loadStories() {
+  req.keys().forEach((filename) => req(filename));
+}
+
+configure(loadStories, module);
+```
+
+You may also need to make Storybook aware of the fact that you're using Babel macros! You can do
+this by declaring `macros` as one of the options in your array of plugins within your babel config.
+
+One example, with a `.babelrc` at the root-level of your repository:
+
+```json
+{
+  "plugins": ["macros"]
+}
+```

